PrioritizedSubPub.js

```diff
@@ -206,12 +206,17 @@
      * @desc private console output. Currently logs by default.
      * @todo: think of good way to toggle logging.
      */
-     function _debugLog() {
+    function _debugLog() {
         var args;
-        if (console && console.log) {
+        if (window.console && console.log) {
```

Comment:
Good point - well it needs to fail gracefully in IE8 as if(console) will throw errors.

So perhaps something more elegant that would fit here is something along the lines of
`hasConsole =  window ? (window.console && window.console.log) : (console && console.log)`

but I leave it for you to decide :)